src/main/java/de/chojo/repbot/util/ContextResolver.java

```diff
@@ -55,4 +64,20 @@
                 .collect(Collectors.toCollection(HashSet::new));
     }
 
+    public Set<Member> getVoiceContext(Message message, int maxAge) {
+        var pastUser = voiceData.getPastUser(message.getAuthor(), message.getGuild(), maxAge, 10);
+        var collect = pastUser.stream()
+                .map(id -> message.getGuild().retrieveMemberById(id).onErrorMap(throwable -> null).complete())
+                .collect(Collectors.toCollection(HashSet::new));
```

Comment:
Collectors.toSet()?

Reply:
I am unsure, since toSet does not guarantee the mutability of this set, but I need to add the voice members to the set.

But I also probably want to use an existing set and switch it to a linked hash set since I want the users in a voice chat first for the button display.

Yeah. Probably this is a better choice.